src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/DynamicContentTypeBuilder.cs

```diff
@@ -53,7 +64,7 @@ public void Build(FieldType contentQuery, ContentTypeDefinition contentTypeDefin
                     continue;
                 }
 
-                if (!(part.PartDefinition.Fields.Any(field => contentFieldProviders.Any(fieldProvider => fieldProvider.HasField(field)))))
+                if (!part.PartDefinition.Fields.Any(field => contentFieldProviders.Any(fieldProvider => fieldProvider.HasField(field))))
```

Comment:
Seems like an unnecessary change

Reply:
This change will solve a warning "Parentheses can be removed [IDE0047]"

Reply:
Sorry, my mistake, I thought it was you who added the brackets